### ISSUE TITLE
protolint was not detected

### ISSUE THREAD

**Issue by reinarudo** [plexsystems/vscode-protolint#42]
Installed via golang and npm. Both ways don't work.
I'm using Windows 11 Version 10.0.26100 Build 26100

<img width="949" alt="Image" src="https://github.com/user-attachments/assets/7d4fd5c9-1de0-4cc5-bbe7-3d9f2b5c3dc9" />

**Comment by WilliamRagstad** [comment]
I get the same problem, even when trying to set the `protolint.path` in settings with either direct binary path of folder to it.
```json
"protolint.path": "C:\\...\\protolint-0.53.0\\protolint.exe",
```
and
```json
"protolint.path": "C:\\...\\protolint-0.53.0",
```

**Comment by Leo-MathGuy** [comment]
Same problem on Mac, protolint is on PATH and tried every setting for path
